src/core/ui.js

```diff
@@ -221,13 +223,14 @@ export const ui = {
     overlay = hyperHTML`<div id='respec-overlay' class='removeOnSave'></div>`;
     const id = `${currentOwner.id}-modal`;
     const headingId = `${id}-heading`;
-    modal = hyperHTML`<div id='${id}' class='respec-modal removeOnSave' role='dialog'>
+    modal = hyperHTML`<div id='${id}' class='respec-modal removeOnSave' role='dialog' aria-labelledby='${headingId}' >
```

Comment:
```suggestion
    modal = hyperHTML`<div id='${id}' class='respec-modal removeOnSave' role='dialog' aria-labelledby='${headingId}' >
      ${closeButton}
```

I think this was the suggestion 👀

Reply:
I had tried that. But a white line was added above the heading line and that is offcourse not desirable
![Screenshot from 2020-01-09 19-18-46](https://user-images.githubusercontent.com/44131389/72072932-eb674e80-3314-11ea-85fb-a63d439481fb.png)

Reply:
Hmm, we might want to do `position: absolute` in that case. @sidvishnoi What do you think?

Reply:
The markup is what we want, so yes, position it with CSS.